tools/deb-arm64/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM arm64v8/debian:buster
+
+WORKDIR /build
+
+ENV TZ=Europe/Belgrade
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+RUN apt update
+RUN apt install -y git gcc g++ cmake automake autotools-dev autoconf m4 zlib1g-dev cmake libtool libssl-dev libpoco-dev devscripts debhelper
+
+COPY make_deb.sh /build
+
+#CMD ["bash", "make_deb.sh"]
```

Comment:
this a comment?

Reply:
The entry point is overridden through the `docker run` command in the `run_arm64.sh`.

https://github.com/Wolkabout/WolkGateway/blob/770e8a68ac88c278c149a61821f743de5650188c/tools/deb-arm64/run-arm64.sh#L34